doc/contributing/contributing-to-documentation.chapter.md

```diff
@@ -29,6 +18,9 @@ As per [RFC 0072](https://github.com/NixOS/rfcs/pull/72), all new documentation
 
 Additional syntax extensions are available, all of which can be used in NixOS option documentation. The following extensions are currently used:
 
```

Comment:
Maybe also mention the [other deviations](https://github.com/NixOS/nixpkgs/blob/8b1253715d80e224908e29a53014b9323933abf7/pkgs/tools/nix/nixos-render-docs/src/nixos_render_docs/md.py#L581-L608):

- HTML is not allowed
- [Examples](https://github.com/NixOS/nixpkgs/blob/be4d19ff1a9a327ae805fdb344470ed6450256fc/doc/stdenv/stdenv.chapter.md?plain=1#L467-L482)
- [Figures](https://github.com/NixOS/nixpkgs/blob/be4d19ff1a9a327ae805fdb344470ed6450256fc/doc/contributing/submitting-changes.chapter.md?plain=1#L217-L221)
- [Footnotes](https://github.com/NixOS/nixpkgs/commit/538b3d1b3c12cc07f00dcec374b16a469b3679ff)